### PR TITLE
fix collabora not working out of the box

### PR DESCRIPTION
**Pull request by szaimen**
Fix https://github.com/nextcloud/all-in-one/issues/424

Signed-off-by: szaimen <szaimen@e.mail.de>

### CODE REVIEW

**Review by juliushaertl — APPROVED** [review]
Should work 👍

**Comment by szaimen** [comment]
Thanks Julius! :)

File: Containers/nextcloud/entrypoint.sh [hunk 1]
```diff
@@ -278,18 +278,22 @@ php /var/www/html/occ config:app:set notify_push base_endpoint --value="https://
...
         php /var/www/html/occ app:enable richdocuments
     else
         php /var/www/html/occ app:update richdocuments
     fi
-    php /var/www/html/occ config:app:set richdocuments wopi_url --value="https://$NC_DOMAIN/"
-    php /var/www/html/occ config:app:set richdocuments public_wopi_url --value="https://$NC_DOMAIN/"
-    # php /var/www/html/occ richdocuments:activate-config
     # Fix https://github.com/nextcloud/all-in-one/issues/188:
     php /var/www/html/occ config:system:set allow_local_remote_servers --type=bool --value=true
+    php /var/www/html/occ config:app:set richdocuments wopi_url --value="http://$COLLABORA_HOST/"
```
Comment (szaimen) [inline Containers/nextcloud/entrypoint.sh:294]: Here might be the host missing

Comment (juliushaertl) [reply Containers/nextcloud/entrypoint.sh:294]: Right, should be :9980 then

Comment (szaimen) [reply Containers/nextcloud/entrypoint.sh]: done

**Pull request merged**